doc/src/build/devnet.md

```diff
@@ -41,32 +41,35 @@ See the [Sui Releases](https://github.com/MystenLabs/sui/releases) page to view
 
 ## Configure Sui client
 
-If you previously ran `sui genesis -f` to create a local network, it created a Sui client configuration file (client.yaml) that connects to `localhost` `http://0.0.0.0:9000`. See [Connect to custom RPC endpoint](#connect-to-custom-rpc-endpoint) to update the client.yaml file.
+If you previously ran `sui genesis` to create a local network, it created a Sui client configuration file (client.yaml) that connects to `localhost` at `http://0.0.0.0:9000`. See [Connect to custom RPC endpoint](#connect-to-custom-rpc-endpoint) to update the client.yaml file.
```

Comment:
Mention that you can also use `sui client new-env` directly instead of updating client.yaml